src/control/go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/armon/go-metrics v0.3.6 // indirect
 	github.com/dustin/go-humanize v1.0.0
 	github.com/fatih/color v1.10.0 // indirect
+	github.com/golang/protobuf v1.5.2
```

Comment:
this shouldn't be added, stick to the golang.org repo

Reply:
Oops, this was added by VSCode trying to be helpful. Removed.